sdk/forms/src/protected-view/iframe/components/view.ts

```diff
@@ -26,16 +26,37 @@ type DisplayValueProps = {
   className?: string;
 };
 
-const DisplayValue = component(
-  ({ value: rootValue, label, path, className, clickToCopy, sendToParent }: DisplayValueProps): Node => {
-    const container = document.createElement('div');
+const DisplayValue = () =>
+  component(({ value: rootValue, label, path, className, clickToCopy, sendToParent }: DisplayValueProps) => {
+    const container = document.createElement('div') as any; // TODO fix type
```

Comment:
Remove `as any`.
You can return `return Object.assign(container, { unmound: () => { ... } })` which will be typed as  `Node & { unmount?: () => void  }`

Reply:
This is an outdated code fragment, eventually instead of `any` I've used `as HTMLDivElement & { unmount?: () => void }`, but I like your solution more. Replaced it with `Object.assign...`